catkin_ws/src/kpr_image_processing/test/TEST_IdCB.cpp

```diff
@@ -0,0 +1,58 @@
+#ifndef TESTING
+	#define TESTING
+#endif
+
+#include "../src/IdCB/IdCB.cpp"
+#include <gtest/gtest.h>
+
+/**
+ * Run all the test cases.
+ */
+int main (int argc, char** argv) {
+	testing::InitGoogleTest(&argc, argv);
+	return RUN_ALL_TESTS();
+}
+
+/**
+ * Tests for to3D() function.
+to3D(cucumber_msgs::Cucumber in, int camera, stereo_msgs::DisparityImage disparity)
+CucumberContainer c = CucumberContainer(x,y,width(cucumber),height,curve);
+ */
+stereo_msgs::DisparityImage CreateDisparity(int f, float T, int step, int width, int vecLength, int dataPoint, float dataPointValue){
```

Comment:
This method is missing its documentation. The comment above is a general one about the tests below